### ISSUE TITLE
add in app

### ISSUE THREAD

**Issue by gentr** [woheller69/Level#25]
today, after I opened the program and switched from the leveler to the meter, the program showed me advertisements for the purchase of pellets and wods in Russian

**Comment by woheller69** [comment]
then I guess it is not installed from F-Droid

**Comment by gentr** [comment]
yes is installed from f-droid, i see advertisements just one time.

**Comment by woheller69** [comment]
can you check the app id in Android settings - Apps - App info for this app.

is it org.woheller69.level  ?

**Comment by gentr** [comment]
advertisement is appearing just one time,  i not see any more, i open many
times, no more advertisements

**Comment by woheller69** [comment]
Btw, my code does not even have internet permission...
